### PR TITLE
Convert ipython notebooks to rst and include in docs

### PR DESCRIPTION
**Pull request by jenshnielsen**

### CODE REVIEW

**Comment by jenshnielsen** [comment]
On top of #146 so that should be merged first. At this stage I am just making sure that travis runs the conversion correctly.

**Comment by jenshnielsen** [comment]
TODO 

- [ ] Make it possible to override kernel from CMake
- [ ] Fix broken notebooks
- [ ] Fix long double issues

**Comment by jenshnielsen** [comment]
retest this please

**Comment by jenshnielsen** [comment]
@garymacindoe Should be ready for review

File: documentation/source/parallel.rst [hunk 1]
```diff
@@ -8,7 +8,24 @@ Running likelihood calculations in parallel
 OpenMP
 ======
 
-TODO Write something about OpenMP 
+HJCFIT will by default be complied with openmp support. The parallelisation
```
Comment (garymacindoe) [inline documentation/source/parallel.rst]: complied -> compiled.

**Pull request merged**